dist/bin/kctf-cluster

```diff
@@ -14,7 +14,27 @@
 # limitations under the License.
 set -Eeuo pipefail
 
-function kctf_cluster_start {
+function create_operator {
+  # Creating CRD, rbac and operator
+  kubectl apply -f "${KCTF_CTF_DIR}/kctf/resources/kctf.dev_challenges_crd.yaml"
+  kubectl apply -f "${KCTF_CTF_DIR}/kctf/resources/rbac.yaml"
+  kubectl apply -f "${KCTF_CTF_DIR}/kctf/resources/operator.yaml"
+
+  OPERATOR_IMAGE=$("${KCTF_BIN}/yq" eval '.spec.template.spec.containers[0].image' "${KCTF_CTF_DIR}/kctf/resources/operator.yaml")
+
+  # The operator needs to create some subresources, e.g. the gcsfuse service account
+  for i in {1..30}; do
+    kubectl get pods --namespace kctf-system -o=jsonpath="{.items[*].status.containerStatuses[?(@.imageID==\"${OPERATOR_IMAGE}\")].ready}" | grep "true" && break
+    if [ "$i" == "30" ]; then
+      echo "Couldn't find a kctf-operator pod with status ready=true and image=\"${OPERATOR_IMAGE}\" after 5 minutes" >&2
+      kubectl get pods --namespace kctf-system -o=yaml >&2
+      exit 1
+    fi
+    sleep 10
```

Comment:
nit:Maybe do 3 seconds (and 100x loop)? It's otherwise a bit boring to see no output.

Reply:
done